invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/DateFormatter.js

```diff
@@ -14,6 +14,10 @@ class DateFormatter extends React.Component {
   render() {
     const { value } = this.props;
 
+    if (!value) {
```

Comment:
shouldn't this be validated by `prop-types`? It has a condition that `value` is a required string.

Reply:
You're right, it's specified as required string, and it's giving a warning, but still renders it. The `end_datetime` value is nullable in db so the `null` is coming from there.